src/EditorFeatures/Core/Host/IWaitContext.cs

```diff
@@ -10,12 +10,14 @@
 
 namespace Microsoft.CodeAnalysis.Editor.Host
 {
+    [Obsolete("You should now use UIThreadOperationStatus, which is a platform supported version of this.")]
     internal enum WaitIndicatorResult
     {
         Completed,
         Canceled,
     }
 
+    [Obsolete("You should now use IUIThreadOperationContext, which is a platform supported version of this.")]
```

Comment:
tagging @minestarks @uniqueiniquity for awareness that this is coming down the pipe.  Should be easy to move off of.  ubt i wanted the obsoletion to be visible to them .

Reply:
Thanks for the heads-up - we have good EA discipline here.